x-pack/plugins/alerting/server/task_runner/create_execution_handler.ts

```diff
@@ -57,12 +57,14 @@ export function createExecutionHandler({
         };
       });
     for (const action of actions) {
-      await executeAction({
-        id: action.id,
-        params: action.params,
-        spaceId,
-        apiKey,
-      });
+      if (actionsPlugin.isActionTypeEnabled(action.actionTypeId)) {
```

Comment:
I feel like we should be doing something in an `else` here, but ... not sure what it might be.  Logging would be too verbose.  I guess we should plan on generating an eventLog record for it ...

Reply:
Yeah, it's a tricky one. Server log may be the only option in the scenario. If the user is downgraded to basic, we won't have the event log enabled due to it being a gold+ feature. Happy to add a TODO comment here or a GH issue, lmk

Reply:
> If the user is downgraded to basic, we won't have the event log enabled due to it being a gold+ feature.

🤦‍♂ 

Ya, TODO it, I think probably log it at this point as well.

Reply:
A warning will now log, code here: https://github.com/elastic/kibana/pull/59699/commits/4c20e4b7c7019e066e31f9e08c200396e84c09be.

In regards of being too noisy, we've discussed that the user can go in the UI to remove that alert action which will stop the logs. If it still is an issue, we can change the message in the future to only show once and something like `Alert "${alertId}" will stop scheduling action "${action.id}" until it is re-enabled`.